qemu/tests/cfg/win_irq_check.cfg

```diff
@@ -84,21 +84,15 @@
                     device_type = "virtio-tablet-pci"
                     input_dev_type_input1 = tablet
     variants:
-        - @default:
         - msi_disable:
             variants:
                 - by_registry:
                     no with_balloon, with_viorng
                     msi_cmd = "reg add "HKLM\System\CurrentControlSet\Enum\%s\Device Parameters\Interrupt Management\MessageSignaledInterruptProperties" /v MSISupported /d %d /t REG_DWORD /f"
                     variants:
-                        - default_queues:
                         - multi_queues:
                             only with_netkvm
                             queues = 4
                 - by_vectors:
-                    only with_viostor, with_netkvm
```

Comment:
If only netkvm need this case, adjust the variants structure only support  netkvm is better.